app/javascript/app/components/collapse/collapse-component.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Button from 'components/button';
+import Icon from 'components/icon';
+import cx from 'classnames';
+import filterIcon from 'assets/icons/filter.svg';
+import ReactDOM from 'react-dom';
+import iconWhite from 'styles/themes/icon/icon-white.scss';
+import iconDefault from 'styles/themes/icon/icon-default.scss';
+
+import styles from './collapse-styles.scss';
+
+class Collapse extends PureComponent {
+  // eslint-disable-line react/prefer-stateless-function
+  renderChildren() {
+    const { contentRef, children, contentClassName } = this.props;
+    const content = (
+      <div className={cx(styles.content, contentClassName)}>{children}</div>
+    );
+
+    return contentRef ? ReactDOM.createPortal(content, contentRef) : content;
+  }
+
+  render() {
+    const { handleToggleCollapse, opened } = this.props;
+
+    return (
+      <div>
+        <Button
+          className={cx({ [styles.open]: opened })}
+          onClick={() => handleToggleCollapse(c => !c)}
```

Comment:
I'd like more to have this function [here](https://github.com/Vizzuality/climate-watch/pull/391/files#diff-181e77e217727451a3abb76c41300e42R4) instead of inside the component.
So to make easier to reuse the collapse component I'd change this function to an onToggleClick callback and I'd do the login in the container.